src/components/productList/AllProducts/index.js

```diff
@@ -2,11 +2,14 @@ import ProductCard from "../../Nuclie/ProductCard";
 
 import { useData } from "../../../Context/stateManage/state-context";
 import productFilter from "../utils/product-filter";
+import filterByRating from "../utils/rating-filter";
 
 export const AllProducts = () => {
   const { state } = useData();
```

Comment:
Please give more meaningful names. As I dont know what this state actually holds.